GermanFilter/sections/whitelist.txt

```diff
@@ -11,6 +11,8 @@
 !###############
 ! https://github.com/AdguardTeam/AdguardFilters/issues/37287
 @@||st.wetteronline.de/dr/*/css/p_*.css
+! https://github.com/AdguardTeam/AdguardFilters/issues/37784
+@@||mydirtyhobby.to^
```

Comment:
which rule breaks the site?

Reply:
That rule `/mydirtyhobby.$domain=~mydirtyhobby.com|~mydirtyhobby.de`. I send report to EasyList team and temporarily added exception.